keps/sig-api-machinery/3903-unknown-version-interoperability-proxy/README.md

```diff
@@ -275,20 +275,48 @@ with a metric.
 
 There could be a large volume of requests for a specific resource which might result in the identified apiserver being unable to serve the proxied requests. This scenario should not occur too frequently, since resource types which have large request volume should not be added or removed during an upgrade -- that would cause other problems, too.
 
-TODO: security / cert stuff.
+We should ensure a definite way for the request flow to complete rather than continuing on indefinitely by proxying the request over and over again.
+
+We need to watch out for inadvertantly putting apiserver IP/endpoint information and the trust bundle control (used to authenticate server while proxying) in user hands (by exposing this information in REST APIs)
```

Comment:
The risk here is that users could modify it and then do a MitM attack?

"in user hands" is a metaphor that isn't very clear.

I would probably rephrase this as "To prevent {failure mode X} we will not {do thing that could lead to that}".

Reply:
Rephrased as suggested.